google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/main/resources/META-INF/services/java.sql.Driver

```diff
@@ -0,0 +1 @@
+com.google.cloud.spanner.jdbc.JdbcDriver
```

Comment:
where is this used?

Reply:
This is used by the Java service loader. See the answer to this question: https://stackoverflow.com/questions/18921352/using-meta-inf-services-for-internal-plumbing-of-driver